### ISSUE TITLE
Accessible heading page does not follow uniform pattern 

### ISSUE THREAD

**Issue by Soyeb01** [AccessibleForAll/AccessibleWebDev#283]
## Issue
Pages theme and templates should be consistent . In **Accessible Heading** page under WCAG Criteria section the links should not be in `<ul>` tag as all the other pages does not follow the same pattern.

## To reproduce
1. Go to **Headings section** page from side navbar.
2. Scroll down to **WCAG Criteria**

### Accessible Headings page
![Screenshot (12)](https://user-images.githubusercontent.com/98175646/233863896-18d0539a-72e7-41f0-88f1-f2aa116f02de.png)

### Accessible Images page or other pages
![Screenshot (13)](https://user-images.githubusercontent.com/98175646/233863900-df709d00-1a64-4a9f-8fdd-447fa578928c.png)

**Comment by EmmaDawsonDev** [comment]
Thanks for spotting this, I had missed it. Let me know if you want to work on the issue

**Comment by Soyeb01** [comment]
Thanks @EmmaDawsonDev and Yes, I would love to work on this issue and contribute.
Please assign me this issue.

**Comment by Soyeb01** [comment]
Hello @EmmaDawsonDev , I have created pull request #284 for this issue. Please review it to merge.
